dev/README_RELEASE_PYTHON_CLIENT.md

```diff
@@ -561,10 +564,10 @@ done
 # Remove old release
 cd ..
 svn rm ${PREVIOUS_VERSION}
-svn commit -m "Release Apache Airflow Python Client ${VERSION} from ${RC}"
+svn commit -m "Release Apache Airflow Python Client ${VERSION} from ${VERSION}${VERSION_SUFFIX}"
 ```
 
-Verify that the packages appear in [airflow](https://downloads.apache.org/airflow/clients/python/)
+Verify that the packages appear in [airflow](https://dist.apache.org/repos/dist/release/airflow/clients/python)
```

Comment:
The downloads link takes a little bit of time to update

Reply:
> The downloads link takes a little bit of time to update

Strange - it should not be (any more). It used to be like that, before ASF Switched to use CDN some 1.5 years ago - since then I never saw a delay. Maybe it's just the matter of Shift-Refresh?

Reply:
But it does not matter. really :)

Reply:
I see . Interesting :)